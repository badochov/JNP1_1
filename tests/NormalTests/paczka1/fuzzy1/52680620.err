Error in line 6: nySbibpe S176 97,8
Error in line 12: nySbibpe A838 24,7
Error in line 5: JKaFpuforG A873 89,5
Error in line 25: JKaFpuforG S176 43,0
Error in line 29: nySbibpe S176 97,4
