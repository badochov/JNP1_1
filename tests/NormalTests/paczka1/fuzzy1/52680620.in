S328 S176 69,6
5QgInCUGdpm S176 78,9
?5QgInCUGdpm
S328 S176 200,4
JKaFpuforG A873 89,5
nySbibpe S176 97,8
Xcw1Zj A838 35,2
?
5QgInCUGdpm S176 261,9
Xcw1Zj A838 19,1
?S328
nySbibpe A838 24,7
uUPxGj A838 69,0
nySbibpe S176 1,3
nySbibpe S176 20,5
JKaFpuforG A838 95,6
S328 A838 38,9
JKaFpuforG A838 236,1
uUPxGj A838 331,9
S328 A838 146,3
?S328
S328 A838 65,0
5QgInCUGdpm A873 45,6
Xcw1Zj A838 83,3
JKaFpuforG S176 43,0

?JKaFpuforG
5QgInCUGdpm A873 98,0
nySbibpe S176 97,4
JKaFpuforG A838 85,9
?S328

?uUPxGj
nySbibpe A838 27,4
nySbibpe A838 87,7
JKaFpuforG A838 31,8
KbyWvf S176 62,8
KbyWvf S176 89,8
KbyWvf S176 98,9
Xcw1Zj A838 90,7