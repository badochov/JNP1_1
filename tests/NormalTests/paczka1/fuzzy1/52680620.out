S328 S 130,8
S176 130,8
S328 S 130,8
S328 A 107,4 S 130,8
JKaFpuforG A 140,5
S328 A 107,4 S 130,8
uUPxGj A 262,9
